src/TRestTrackLineAnalysisProcess.cxx

```diff
@@ -0,0 +1,252 @@
+/*************************************************************************
+ * This file is part of the REST software framework.                     *
+ *                                                                       *
+ * Copyright (C) 2016 GIFNA/TREX (University of Zaragoza)                *
+ * For more information see http://gifna.unizar.es/trex                  *
+ *                                                                       *
+ * REST is free software: you can redistribute it and/or modify          *
+ * it under the terms of the GNU General Public License as published by  *
+ * the Free Software Foundation, either version 3 of the License, or     *
+ * (at your option) any later version.                                   *
+ *                                                                       *
+ * REST is distributed in the hope that it will be useful,               *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          *
+ * GNU General Public License for more details.                          *
+ *                                                                       *
+ * You should have a copy of the GNU General Public License along with   *
+ * REST in $REST_PATH/LICENSE.                                           *
+ * If not, see http://www.gnu.org/licenses/.                             *
+ * For the list of contributors see $REST_PATH/CREDITS.                  *
+ *************************************************************************/
+
+//////////////////////////////////////////////////////////////////////////
+/// The TRestTrackLineAnalysis process is meant to analyze lienar tracks,
+/// such as alpha tracks. It should retreive the information after performing
+/// TRestTrackLinearization process, for the time being only works when
+/// XZ and YZ are present in the readout. Moreover, the hits have to be stored
+/// following the minimum path. It assumes that only one alpha track
```

Comment:
Probably it could be generalised to a multi track event by picking up the track with the maximum energy reusing one of these methods:

```
    TRestTrack* GetMaxEnergyTrackInX();
    TRestTrack* GetMaxEnergyTrackInY();

    TRestTrack* GetMaxEnergyTrack(TString option = "");
```

Reply:
Done